eng/pipelines/runtime-official.yml

```diff
@@ -0,0 +1,98 @@
+trigger:
```

Comment:
Do we really need a dedicated `runtime-official.yml`? Should we instead use `runtime.yml` and condition configurations and matrix based on `isOfficialBuild`? 

Usually maintaining multiple .yml files that define different matrixes is kind of tricky because a configuration can be forgotten in some yml, such as triggers and stuff like that.

Reply:
> Should we instead use `runtime.yml` and condition configurations and matrix based on `isOfficialBuild`?

I think we should. It will take some extra time to condition the configurations and list of platforms correctly, I don't think we need to do it with the initial implementation, but I'm open to it if you think it's critical.

Reply:
Sounds good. I think once we have the single pipeline landed we can merge these two and try to do it as clean as possible, but having these 2 separated now makes sense, specially because of all the changes it will suffer in the coming days.

Reply:
Cool, added this to the list in https://github.com/dotnet/runtime/issues/749 to keep track of it.